src/frontend/screens/Settings/sections/LogSettings/index.css

```diff
@@ -39,7 +39,14 @@
   display: flex;
   flex-direction: column;
   gap: var(--space-md);
-  max-width: 50vw;
+  width: 100vw;
+  max-width: 900px;
+}
+
+@media screen and (max-width: 1024px) {
+  .logs-wrapper.game-log {
+    width: calc(80vw - 24px);
+  }
```

Comment:
the logs dialog looked wrong when shrinking the screen

![image](https://github.com/user-attachments/assets/fafae51c-a469-4b90-931a-29c36468c779)

and now:
![image](https://github.com/user-attachments/assets/32956232-75e0-46b0-a7a6-e78c54f93d10)


(it still look a bit weird with really narrow screens, but I don't think it's an expected use case, other things break in the UI at that point)